### DISCUSSION TITLE
Status of Quarkus Long Term Support

### DISCUSSION THREAD

**Discussion by tyborch** [quarkusio/quarkus#34556] [Q&A] [answered]
As stated on the [Quarkus 3](https://quarkus.io/quarkus3/) landing page.
"We often get asked about a Quarkus LTS version. The community asked, so we’ll deliver. Starting with Quarkus 3.2, we will introduce the first Quarkus LTS version. We will share more information as we get closer to releasing Quarkus 3.2."

With the release of Quarkus 3.2.0.Final on 07/05 I was wondering where we are at with said Long Term Support.

Thanks!

**Comment by maxandersen** [comment, marked as answer]
Post will go out tomorrow.

**Comment by maxandersen** [comment]
here  you go https://quarkus.io/blog/lts-releases/
